deepchecks/tabular/checks/train_test_validation/train_test_feature_drift.py

```diff
@@ -229,11 +232,18 @@ def run_logic(self, context: Context) -> CheckResult:
             if self.sort_feature_by == 'feature importance' and feature_importance is not None:
                 features_order = [feat for feat in features_order if feat in train_dataset.features]
                 columns_order = features_order[:self.n_top_columns]
+            elif self.sort_feature_by == 'drift + importance' and feature_importance is not None:
+                feature_columns = [feat for feat in features_order if feat in train_dataset.features]
+                feature_columns.sort(key=lambda col: values_dict[col]['Drift score'] + values_dict[col]['Importance'],
+                                     reverse=True)
+                columns_order = feature_columns[:self.n_top_columns]
             else:
                 columns_order = sorted(values_dict.keys(), key=lambda col: values_dict[col]['Drift score'],
                                        reverse=True)[:self.n_top_columns]
 
             sorted_by = self.sort_feature_by if feature_importance is not None else 'drift score'
+            if sorted_by == 'drift + importance':
```

Comment:
if self.sort_feature_by == 'drift + importance' and feature_importance is None, then you will print that it was sorted by "drift + importance" when it was actually sorted by drift score.

Reply:
No because if `feature_importance` is none then the line above would have changed sorted_by  to ''drift score'.
It's actually covered by the test.